zjit/src/hir.rs

```diff
@@ -1020,6 +1020,30 @@ impl Insn {
 pub struct InsnPrinter<'a> {
     inner: Insn,
     ptr_map: &'a PtrPrintMap,
+    iseq: Option<IseqPtr>,
+}
+
+/// Get the name of a local variable given iseq, level, and ep_offset.
+/// Returns
+/// - `:name, ` if iseq is available and name is a real identifier,
+/// - `<anon>, ` for anonymous locals.
+/// - empty string if iseq is not available.
```

Comment:
nit: does not return empty anymore if iseq is not available--returns none. pls fix comment